lib/api/core/janitor.js

```diff
@@ -362,6 +465,73 @@ class Janitor {
     process.exit(0);
   }
 
+  /**
+   * Load a json file into a javascript object.
+   * It's better than a simple require because we asynchronously read the file
+   * and we avoid RCE if a javascript file is provided instead of JSON
+   *
+   */
+  _loadJson (filePath) {
+    return new Bluebird((resolve, reject) => {
+      fs.readFile(filePath, 'utf8', (error, rawData) => {
+        if (error) {
+          return reject(error);
+        }
+
+        try {
+          const data = JSON.parse(rawData);
+
+          resolve(data);
+        }
+        catch (e) {
+          reject(e);
+        }
+      });
+    });
+  }
+
+  _createSecurity (action, objects) {
+    if (! objects) {
+      return Bluebird.resolve();
+    }
+
+    return Bluebird.map(Object.keys(objects), id => {
+      const request = new Request({
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Group all shorthand properties at either the beginning or end of this object declaration. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS3499)